packages/codecs/src/ipld.ts

```diff
@@ -0,0 +1,49 @@
+import { CID } from 'multiformats/cid'
+import { type Context, Type } from 'codeco'
+
+/**
+ * Passthrough codeco codec for CID.
+ */
+export const cid = new Type<CID, CID, unknown>(
```

Comment:
does it make sense to have the ipld ones here? didnt see where they were used

Reply:
oh saw what you said before, since js-ceramic depends on dids dependencies

Reply:
Yes we need it here as well to define the DagJWS codec: https://github.com/ceramicnetwork/js-did/pull/153/files#diff-381c167d850c6491abefda0be2d4d6616083316c929a147f759358602a031592R14